src/controller/java/templates/ClusterInfo-java.zapt

```diff
@@ -250,13 +250,20 @@ public class ClusterInfoMapping {
  public Map<String, ClusterInfo> getClusterMap() {
     Map<String, ClusterInfo> clusterMap = new HashMap<>();
     getCommandMap(clusterMap);
-    getReadAttributeMap(clusterMap);
+    Map<String, Map<String, InteractionInfo>> readAttributeMap = getReadAttributeMap();
+    combineCommand(clusterMap, readAttributeMap);
```

Comment:
Like we discussed earlier, could we have something like this?
```
Map<String, ClusterInfo> clusterMap = initializeClusterMap();
Map<String, Map<String, InteractionInfo>> commandMap = getCommandMap(clusterMap);
combineCommand(clusterMap, commandMap);
Map<String, Map<String, InteractionInfo>> readAttributeMap = getReadAttributeMap();
combineCommand(clusterMap, readAttributeMap);
```
We want `getCommandMap` and `readAttributeMap` to be consistent.